airbyte/_processors/sql/snowflakecortex.py

```diff
@@ -107,9 +107,10 @@ def _get_column_list_from_table(
         This is overridden due to lack of SQLAlchemy compatibility for the
         `VECTOR` data type.
         """
+        normalized_table_name = self.normalizer.normalize(table_name)
```

Comment:
updated to use self.normalize which replaces spaces and hyphens with underscore..

Reply:
Nice - yeah, this is great. I will also look to see why it was not already normalized. Whenever we pass a "table_name" arg, it should be already normalized. But no reason to block you if this is working.